app/Policies/UserPolicy.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Policies;
+
+use App\Role;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class UserPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether one user can change the Repair Directory role of another to a specific value.
+     *
+     * @param  \App\User  $user
+     * @param  \App\User  $user
+     * @param  int  $role
+     * @return mixed
+     */
+    public function canChangeRepairDirRole(User $perpetrator, User $victim, int $role)
```

Comment:
I think the naming convention would be to have the method called 'changeRepairDirRole' (as when we call in a blade template, we might use @can('changeRepairDirRole', ...)

Reply:
Will change.